### PR TITLE
Stage -> Dev Sync

### PR DESCRIPTION
**Pull request by anthonypetersen**
Backing some changes needed in stage to dev

### CODE REVIEW

**Review by JoeArmani — APPROVED** [review]
backing in to sync, already active in stage. looks good

File: utils/promis.js [hunk 1]
```diff
@@ -24,6 +22,7 @@ const processPromisResults = async (uid) => {
 
     const forms = Object.keys(promisConfig);
     const scoresPayload = {};
+    const scoresPromises = [];
```
Comment (we-ai) [inline utils/promis.js:25]: One promise may not need an array to hold it.

**Review by we-ai — APPROVED** [review]

File: utils/promis.js (hunk 1 above)
Comment (anthonypetersen) [reply utils/promis.js:25]: there are many promises being generated on the `for` loop starting on line 27

Comment (anthonypetersen) [reply utils/promis.js:25]: There are many promises being generated in the

**Pull request merged**